openmrs/i18n/home/locale_en.json

```diff
@@ -11,5 +11,35 @@
     "MODULE_LABEL_PROGRAMS_KEY": "Programs",
     "MODULE_LABEL_IMPLEMENTER_INTERFACE_KEY": "Implementer Interface",
     "MODULE_LABEL_ATOMFEED_CONSOLE_KEY": "AtomFeed Console",
-    "MODULE_LABEL_APPOINTMENT_SCHEDULING_KEY": "Appointment Scheduling"
+    "MODULE_LABEL_APPOINTMENT_SCHEDULING_KEY": "Appointment Scheduling",
+    "HOME_LOCATION_HOSPITAL":  "Hopital de la Cité Verte",
+    "HOME_LOCATION_HOME_LOCATION_DESK":	"Registration Desk",
+    "HOME_LOCATION_OPD":	"OPD",
```

Comment:
@prashantsable1991 I would suggest removing the tabulation after ':' (and use a simple space instead) to keep the format consistent across all the locale files